Core/src/Material/SurfaceMaterialMapper.cpp

```diff
@@ -285,8 +285,7 @@ void Acts::SurfaceMaterialMapper::mapMaterialTrack(
         currentPos, rmIter->materialProperties, currentPathCorrection);
     touchedMapBins.insert(MapBin(&(currentAccMaterial->second), tBin));
     ++assignedMaterial[currentID];
-    // Update the material interaction with the associated surface and direction
-    rmIter->direction = mTrack.first.second.normalized();
```

Comment:
Why is the direction dropped ?

Reply:
This used to be use to add the direction information to the material interaction. Now we directly store it at the geantino recording step so this is not needed anymore (it should have been removed a while ago but I forgot).

Reply:
I see.